src/OrchardCore/OrchardCore.Users.Abstractions/Events/IAccountEvents.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Users.Events
+{
+    public interface IAccountEvents
```

Comment:
Have you looked at the events we have in O1 to see if you are missing any?

Reply:
In O1 we have more events;
LoggedOut
AccessDenied

And also for the other events we have more. Do you want me to include those as well? I've now implemented what I need.